src/main/kotlin/com/jay/racingcar/domain/Winners.kt

```diff
@@ -0,0 +1,14 @@
+package com.jay.racingcar.domain
+
+class Winners(racingCars: RacingCars) {
+    private val winners: List<RacingCar>
+
+    init {
+        val sortedRacingCarsByPosition = racingCars.racingCars.sorted().groupBy { it.getPosition() }.toMap()
+        winners = sortedRacingCarsByPosition.values.first()
+    }
+
+    fun getWinners(): List<RacingCar> = winners
+
+    fun getWinnerNames(): List<String> = winners.map { it.getNameValue() }.toList()
```

Comment:
```suggestion
    fun getWinnerNames(): List<String> = winners.map { it.getNameValue() }
```
toList 함수를 호출하여 방어적 복사를 하는 의도 너무 좋습니다. 👍 
다만, 모든 컬렉션 함수들(map, filter 등)은 호출 시 매번 새로운 컬렉션을 만듭니다.
해서 toList 함수를 호출하지 않으셔도 됩니다. 😄

Reply:
앗 그렇군요 감사합니다!